strutil/cmdline.go

```diff
@@ -157,3 +158,29 @@ func KernelCommandLineSplit(s string) (out []string, err error) {
 	}
 	return out, nil
 }
+
+// GetKernelCommandLineKeyValue splits the given kernel command line string into
+// individual parameters using KernelCommandLineSplit, then matches for key
+// value pairs and returns the value of the specified key.
+func GetKernelCommandLineKeyValue(s string, key string) (string, error) {
```

Comment:
the resplitting that this as is does is probably not expensive but a bit odd. My preference would be to turn this into

KernelCommandLineKeyValue(key ...string) (map[string]string, err)

so this would also do the reading,  we would have add/move (there is one in boot now) osutil.MockProcCmdline here for tests.

Reply:
this is done now